src/commands/run.ts

```diff
@@ -80,7 +80,7 @@ export default class Run {
     console.info('Scan duration %ds.', hrend[0]);
 
     if (scanResult.shouldExitOnEnd) {
-      process.exit(1);
+      process.exit(cmd.ci ? 0 : 1); // could be written as +!cmd.ci but I'm not here to show off
```

Comment:
I'd rather have the condition in the previous condition on a line above. There is more feedback that exit code 1 does not make sense. Please completely remove the exit code 1 and update the documentation.

Reply:
Discussed with @prokopsimek and we'll move this change to another PR.